tasks/licenses.py

```diff
@@ -79,9 +79,21 @@ def get_licenses_list(ctx):
 
 def licenses_csv(licenses):
     licenses.sort(key=lambda lic: lic["package"])
-
+    def is_valid_quote(copyright):
+        stack = []
+        quotes_to_check = ["'",'"']
+        for c in copyright:
+            if c in quotes_to_check:
+                if len(stack) !=0 and stack[-1] == c:
+                    stack.pop()
+                else:
+                    stack.append(c)
+        return len(stack) == 0
     def fmt_copyright(lic):
-        copyright = ' | '.join(sorted(lic['copyright']))
+        # discards copyright with invalid quotes to ensure generated csv is valid
+        copyright = [f_copyright for f_copyright in lic['copyright'] if is_valid_quote(f_copyright)]
```

Comment:
It might be good to have something printed whenever a copyright is discarded so that people can understand why it's not present in the `.csv` file. It could also display a message saying you can override the copyright by using the `.copyright-overrides.yml` file if the user wants to fix the bad-quoted package. WDYT?

Reply:
Thanks for you feedback! I just added a message when a copyright is discarded because of invalid quotes. If all the copyright of a given lib are discarded it will still appear in the license csv file, but with UNKNOWN as a copyright. Libs with UNKNOWN copyright are handled in the go.generate-licenses task and they will need to be manually fixed.